deploy/istio-workspace/role.yaml

```diff
@@ -50,3 +50,9 @@ rules:
       - '*'
     verbs:
       - '*'
+  - apiGroups:
+      - networking.istio.io
```

Comment:
@aslakknutsen maybe we should align our `apiGroup` now to `sessions.istio.io`. WDYT?

Reply:
Technically we're not part of the istio project. Wonder if we should change it to something like maistra.io instead?

Reply:
I meant `maistra`. That was pre-coffee

Reply:
https://github.com/Maistra/istio-workspace/issues/208